package.json

```diff
@@ -44,5 +46,10 @@
     "react-router": "^2.4.0",
     "sinon": "^1.17.2",
     "sinon-chai": "^2.8.0"
+  },
+  "publishr": {
+    "dependencies": [
```

Comment:
This looks correct if the goal is to move the `builder` and `builder-victory-component` dependencies to devDependencies in the npm repo.